src/modules/systemlib/mixer/mixer.h

```diff
@@ -184,6 +184,15 @@ class __EXPORT Mixer
 	 */
 	virtual void			groups_required(uint32_t &groups) = 0;
 
+	/**
+	 * @brief      Update slew rate parameter. This tells the multicopter mixer
+	 *             the maximum allowed change of the output values per cycle.
+	 *
+	 * @param[in]  slew_rate_max  The maximum slew rate.
+	 */
+	virtual void 			update_slew_rate(float slew_rate_max) {};
```

Comment:
Is this meant to be just an interface? Because it doesn't do what it says it does.